LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+                            Preamble
```

Comment:
- [ ] Kindly replace this license with the MIT license that Microverse provided and fill it with your information:
https://github.com/microverseinc/readme-template/blob/master/MIT.md